CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
 ## [2.8.0] - Unreleased
 
+- Removed indexedDB support from UserData [ticket](https://pbskids.atlassian.net/browse/SR-66)
```

Comment:
Generally we don't link out to tickets in SpringRoll because this is public and folks won't have access to Jira